release-team/role-handbooks/docs/README.md

```diff
@@ -54,7 +56,7 @@ In addition to the time requirements above, a Docs Lead must:
 - Have the ability to `/approve` PRs, so must be a member of [sig-docs-maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers)
 
```

Comment:
Hi Jim, should there be a link or a short description on how to request for sig-docs-maintainers membership?

Reply:
That would be great! Do you want to suggest how to phrase it and link your PR and I'll add it to the PR?

Reply:
What do you think about the following,
- Have the ability to `/approve` PRs, so must be a member of [sig-docs-maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers). Membership can be requested by creating a [PR] (https://github.com/kubernetes/website/pull/20351) against `master` branch.